contrib/kcp.code-workspace

```diff
@@ -2,12 +2,16 @@
 	"folders": [
 		{
 			"name": "kubernetes",
-			"path": "../../../../k8s.io/kubernetes"
+			"path": "../../../go/src/k8s.io/kubernetes"
```

Comment:
Why changing this ? Is it specific to your environment ?

Reply:
Yes.. when I rebase I will drop it :)